.github/workflows/ci.yaml

```diff
@@ -207,10 +207,13 @@ jobs:
             - name: Mount bazel caches
               uses: actions/cache@v4
               with:
+                  # Cache the --dist_cache and --repository_cache directories (see ci.bazelrc)
+                  # and the default xdg cache (see XDG_CACHE_HOME docs https://bazel.build/remote/output-directories#layout)
```

Comment:
this comment is a bit confusing, since we're not seeing XDG_CACHE_HOME; maybe just say it's the default output root?

Reply:
True enough, I'll just mention the bazel-output and bazelisk while keeping the link.